ert3/exceptions/_exceptions.py

```diff
@@ -0,0 +1,13 @@
+import sys
+
+class Error(Exception):
+    """Base class for exceptions in this module."""
+    pass
+
+
+class IllegalWorkspaceOperation(Error):
+    """Exception raised when user tries to perform illegal workspace operation, for example, initialise it twice.
+    """
+    def __init__(self, message):
+        self.args = "{}".format(message),
+        sys.exit(self)
```

Comment:
I would suggest to leave the `sys.exit` to whoever catches the exception.

Reason for this is that you really don't want to control the program flow from _within_ an exception.